app/views/users/show.html.erb

```diff
@@ -1,2 +1,20 @@
 <h1><%= @user.name %></h1>
-<!-- Other user details -->
+<% if @user.photo.present? %>
+  <%= image_tag @user.photo %>
+<% else %>
+  <p>No photo available</p>
+<% end %>
+<p><%= @user.bio %></p>
+<p>Number of posts: <%= @user.posts.count %></p>
+
+<% @user.posts.each_with_index do |post, index| %>
+  <h2>Post #<%= index + 1 %> by <%= post.author.name %></h2>
+  <p><%= post.content %></p>

```

Comment:
- [ ] Kindly make use of the model methods you created in the previous milestone to render only three recent posts for a user. You should only show (render) all the posts when a user clicks `the See all posts` button.
- [ ] Kindly fix the error on line 12, I'm sure it should be `post.text`, instead of `post.content`.

![show-page](https://github.com/TayyabKM/blog-app/assets/53124548/028a7b73-d4b2-4797-a5f1-ec4121399afb)